src/main/java/io/qameta/allure/bamboo/AllureDownloader.java

```diff
@@ -68,6 +73,20 @@
         return Optional.empty();
     }
 
+    private void addExecutablePermissions(Path allureHome) {
+        Set<PosixFilePermission> perms = new HashSet<>();
+        perms.add(PosixFilePermission.OWNER_EXECUTE);
+        perms.add(PosixFilePermission.OWNER_READ);
+        perms.add(PosixFilePermission.OWNER_WRITE);
```

Comment:
no need to set read or write permissions to execute script. Only execute should be just enough